reporting/.gitignore

```
@@ -1,3 +1,4 @@
 example/index.bundle.js
 example/content.bundle.js
 example/manifest.json
+scenariors/*
```

Looks like a typo.
```suggestion
scenarios/*
```

Though I wonder if they are intended to be committed to this project.